recipe/meta.yaml

```diff
@@ -10,9 +10,13 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 2
   script: mkdir -p "$PREFIX"/include && cp -r cub "$PREFIX"/include  # [not win]
 
+requirements:
+  run:
+    - thrust {{ version }}.*
```

Comment:
Do we intend to make `thrust` a requirement here or merely align versions? If the former, this seems fine. If the latter, we can use [`run_constrained`]( https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained ) to ensure that this version constraint is met when the user installs `thrust`, but doesn't actually require `thrust` to be installed.

Reply:
Thrust is a dependency of CUB.

Reply:
I am curious if we should add conda's compilers here?